packages/graphql-mocks/src/highlight/utils/unique.ts

```diff
@@ -1,12 +1,19 @@
 import { Reference } from '../types';
-import { isEqual } from './is-equal';
 
 export function unique(fieldReferences: Reference[]): Reference[] {
+  const stringified = fieldReferences.map((reference): string => {
+    return Array.isArray(reference) ? `!!!${reference[0]}:::${reference[1]}` : reference;
+  });
+
   const uniques: Reference[] = [];
+  // automatically dedupe strings using a `Set`
+  new Set(stringified).forEach((string) => {
```

Comment:
This is way faster. Something like where strings are put in a set and then pulled back out. Since named graphql types have reserved characters some set of these can be used as a header to mark it's a [type, field] reference, and to use as the separator value.

Reply:
Essentially, remove all usages of `isEqual`, it's a constant slowdown with big schemas. It was previously written to avoid a previous slowdown in https://github.com/graphql-mocks/graphql-mocks/pull/227

Reply:
Done in #274